packages/errors/src/message-formatter.ts

```diff
@@ -19,7 +19,7 @@ function encodeValue(value: unknown): string {
     } else {
         return encodeURIComponent(
             String(
-                Object.getPrototypeOf(value) === null
+                value != undefined && value != null && Object.getPrototypeOf(value) === null
```

Comment:
This is the canonical ‘null or undefined’ check (with the `!=` instead of the `!==`):
```suggestion
                value != null && Object.getPrototypeOf(value) === null
```